lib/subghz/protocols/dickert_mahs.c

```diff
@@ -0,0 +1,487 @@
+#include "dickert_mahs.h"
+
+#include "../blocks/const.h"
+#include "../blocks/decoder.h"
+#include "../blocks/encoder.h"
+#include "../blocks/generic.h"
+#include "../blocks/math.h"
+
+#include <furi.h>
+#include <furi_hal.h>
+#include <furi_hal_rtc.h>
+
+/*
+ * Help
+ * 
+ *
+ */
+
+#define TAG "SubGhzProtocolDicketMAHS"
+#define DICKERT_MAHS_12_COUNT_BIT 12
+#define DICKERT_MAHS_24_COUNT_BIT 24
+#define PRASTEL_COUNT_BIT 25
+#define PRASTEL_NAME "Prastel"
+#define AIRFORCE_COUNT_BIT 18
+#define AIRFORCE_NAME "Airforce"
+
+#define MINUS 0x0
+#define ZERO 0x1
+#define PLUS 0x3
+
+Storage* storage;
```

Comment:
Such globals and the whole idea of logging into a file from protocol code are a bad idea. Please remove that.

Reply:
Removed as requested